### ISSUE TITLE
`slideWidth` Doesn't get set correctly.

### ISSUE THREAD

**Issue by marbuser** [SSENSE/vue-carousel#357]
## Bug Report

**Current Behavior**
Creating a carousel with 4 slides with the config;
```
ref="carousel"
:autoplay="false"
:perPageCustom="[[320, 1], [768, 2]]"
:paginationEnabled="false"
:mouseDrag="false"
```
Maybe 20% of the time in the browser is works as intended. The other 80% the `slideWidth` gets set to 32 (When it should be 300).

As soon as you interact with the carousel it fixes, but that's obviously not the idea solution.

I have no idea what is causing this and it doesn't really make much sense.

**Input Code and steps to reproduce**
```js
            <carousel
              ref="carousel"
              :autoplay="false"
              :perPageCustom="[[320, 1], [768, 2]]"
              :paginationEnabled="false"
              :mouseDrag="false"
              class="carousel"
            >
              <slide
                v-for="slide in products"
                :key="slide.image"
                class="slide"
              >
                <img
                  :src="image"
                  alt=""
                  @click="lightboxSpawn(image)"
                />
              </slide>
            </carousel>
```

**Expected behavior/code**
The `slideWidth` should be calculated correctly as when it isn't, it sets the `flex-basis` to something really small which causes the slides to be very small. 

**Babel Configuration (.babelrc, package.json, cli command)**

```js
module.exports = {
  presets: ['@vue/app'],
};
```

**Environment**
- Babel version(s): Not sure. Using the Vue CLI Plugin.
- Node/npm version: Node 11
- OS: OSX 10.14.2

**Comment by marbuser** [comment]
Any update on this? It's been almost a month and not a single reply. :/

**Comment by quinnlangille** [comment]
Hey @marbuser, I missed the original post! Sorry about not replying. This is an open issue (you can find more details here #233) and we're working on a a fix currently #274. Unfortunately, it's not an easy issue and has a ton of side effects. The refactor is planned as part of v1, but in the mean time you can find a few workarounds in #233! Lemme know if you have any questions about it - I'm going to close this, feel free to direct any further comments to #22 :octocat:
